### PR TITLE
Add full-width input title

### PR DESCRIPTION
**Pull request by dimaostrov**
Updated issue 73 for dialog bar to have a inputable title

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "paper-dropdown-menu": "PolymerElements/paper-dropdown-menu#^2.1.0",
     "paper-fab": "PolymerElements/paper-fab#^2.1.0",
     "paper-icon-button": "PolymerElements/paper-icon-button#^2.2.0",
-    "paper-input": "PolymerElements/paper-input#^2.2.3",
+    "paper-input": "PolymerElements/paper-input#2.0-preview",
```
Comment (CSchmitz81) [inline bower.json]: Do we really need to downgrade the version?

File: src/ctree-new-element-screen/ctree-new-element-screen.html [hunk 2]
```diff
@@ -74,27 +75,40 @@
...
       }
+      .input {
+        width: 100%;
+      }
     </style>
 
     <!--<ctree-import id="importPage" href="/src/ctree-new-element-screen/ctree-[[page]]-page.html" loading="{{_importingPage}}"></ctree-import>-->
 
     <div id="container">
-      <paper-toolbar class="toolbar" style="background-color: [[headerColor]];">
```
Comment (CSchmitz81) [inline src/ctree-new-element-screen/ctree-new-element-screen.html]: Do we really need to remove this class?

File: src/ctree-new-element-screen/ctree-new-element-screen.html [hunk 3]
```diff
@@ -74,27 +75,40 @@
...
+            </paper-item>
+            <paper-item>
+              <iron-icon src="/images/app-icon-32.png"></iron-icon> Seed
+            </paper-item>
+            <paper-item>
+              <iron-icon src="/images/app-icon-32.png"></iron-icon> Seed
+            </paper-item>
+            <paper-item>
+              <iron-icon src="/images/app-icon-32.png"></iron-icon> Seed
+            </paper-item>
```
Comment (CSchmitz81) [inline src/ctree-new-element-screen/ctree-new-element-screen.html]: I tend to prefer these inlined like they were, but not a big deal

File: src/ctree-new-element-screen/ctree-new-element-screen.html [hunk 4]
```diff
@@ -74,27 +75,40 @@
...
           </paper-listbox>
         </paper-dropdown-menu>
         <span slot="top" class="title">[[element.title]]</span>
+        <paper-input slot="top" class="input"></paper-input>
         <paper-icon-button slot="top" id="close" icon="close" alt="close" on-tap="_closeClicked" hidden$="[[noclose]]"></paper-icon-button>
         <paper-tooltip slot="top" for="close" offset="-8">Close</paper-tooltip>
-        <paper-tooltip slot="top" for="typeIcon" position="right" offset="8" animation-delay="0" class="type-description">[[element.type.name]]: [[element.type.description]]</paper-tooltip>
+        <paper-tooltip slot="top" for="typeIcon" position="right" offset="8" animation-delay="0" class="type-description">[[element.type.name]]:
+          [[element.type.description]]</paper-tooltip>
```
Comment (CSchmitz81) [inline src/ctree-new-element-screen/ctree-new-element-screen.html]: Adding a newline here may change the way it appears.

**Comment by dimaostrov** [comment]
Yeah, rookie mistake, I should of cleaned my branch and only use the method you showed me, so there's a bunch of junk that I was experimenting with trying to figure the darn slot property!

Let me remake it and I'll make another clean PR.

**Comment by CSchmitz81** [comment]
deleted branch, changes merged in #95

**Pull request closed without merging**